InterviewQuestionsTests/Chapters/BitInsertions/BitInsertions.Tests.cs

```diff
@@ -0,0 +1,20 @@
+using InterviewQuestions.Chapters.BitInsertion;
+
+namespace InterviewQuestionsTests.Chapters.BitInsertion
+{
+    [TestClass] 
+    public class BitInsertionTest
+    {
+        [TestMethod]
+        public void BitInsertionMethod()
```

Comment:
could you change this test name to something like "bitInsertionTest" ?

Reply:
done, you can look here 7758d34a7b1aa87985bb842eb81d5af158e2e2e9